### ISSUE TITLE
Builder Header Failed Save Example: Tooltips not read

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#2298]
In the Builder Header Failed Save example http://design-system-react-components.herokuapp.com/?path=/story/sldsbuilderheader--failed-save , the status text, nor the icons have tab index and therefore the tooltip cannot be read by screen readers, etc.

**Comment by interactivellama** [comment]
@garygong FYI, on the accessibility of the BuilderHeader save and error status as it is in the example currently.
